VATSpy.dat

```diff
@@ -17889,17 +17889,23 @@ EDMM-G|Muenchen ACC (Gera) - Muenchen|EDMM_G|EDMM-G
 EDMM-G|Muenchen ACC (Gera) - Muenchen|EDMM_M|EDMM-G
 EDMM-G|Muenchen ACC (Gera) - Muenchen|EDWW_B|EDMM-G
 EDWW|Bremen||EDWW
-EDWW-B|Bremen ACC (Boerde) - Bremen|EDWW_F|EDWW-B
-EDWW-B|Bremen ACC (Boerde) - Bremen|EDWW_B|EDWW-B
-EDWW-B|Bremen ACC (Boerde) - Bremen|EDWW_K|EDWW-B
-EDWW-B|Bremen ACC (Boerde) - Bremen|EDWW_M|EDWW-B
-EDWW-A|Bremen ACC (Aller) - Bremen|EDWW_A|EDWW-A
-EDWW-A|Bremen ACC (Aller) - Bremen|EDWW_H|EDWW-A
-EDWW-A|Bremen ACC (Aller) - Bremen|EDWW_D|EDWW-A
-EDWW-A|Bremen ACC (Aller) - Bremen|EDWW_R|EDWW-A
-EDWW-A|Bremen ACC (Aller) - Bremen|EDWW_S|EDWW-A
-EDWW-A|Bremen ACC (Aller) - Bremen|EDWW_E|EDWW-A
-EDWW-A|Bremen ACC (Aller) - Bremen|EDWW_W|EDWW-A
+EDWW-B|Bremen Radar (Boerde) - Bremen|EDWW_F|EDWW-B
+EDWW-B|Bremen Radar (Boerde) - Bremen|EDWW_B|EDWW-B

```

Comment:
There is two entries for EDWW_B, this one and ``EDMM-G|Muenchen ACC (Gera) - Muenchen|EDWW_B|EDMM-G``

Not sure if this would even work..